changes/en-us/2.x.md

```diff
@@ -22,6 +22,7 @@ Add changes here for all PR submitted to the 2.x branch.
 - [[#6091](https://github.com/seata/seata/pull/6091)] Optimizing the method of obtaining the tc address during raft authentication
 - [[#6098](https://github.com/seata/seata/pull/6098)] optimize the retry logic in the acquireMetadata method
 - [[#6034](https://github.com/seata/seata/pull/6034)] using namespace from command line when deployment with helm charts
+- [[#6125](https://github.com/seata/seata/pull/6125)] unbind xid in TransactionTemplateTest
```

Comment:
For pr's involving unit tests, I suggest that they be grouped together in `test`.

Reply:
done